internal/proxy/rate_limit_interceptor.go

```diff
@@ -41,7 +41,7 @@ func RateLimitInterceptor(limiter types.Limiter) grpc.UnaryServerInterceptor {
 	return func(ctx context.Context, req any, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
 		dbID, collectionIDToPartIDs, rt, n, err := getRequestInfo(ctx, req)
 		if err != nil {
-			log.RatedWarn(10, "failed to get request info", zap.Error(err))
+			log.Warn("failed to get request info", zap.Error(err))
```

Comment:
why?

Reply:
Under normal circumstances, there should not be any error.